ext/mysqli/mysqli.stub.php

```diff
@@ -547,225 +547,225 @@ function mysqli_connect_errno(): int {}
+function mysqli_stmt_bind_result(mysqli_stmt $stmt, mixed &...$vars): bool {}
 
-function mysqli_stmt_close(mysqli_stmt $mysql_stmt): bool {}
+function mysqli_stmt_close(mysqli_stmt $stmt): bool {}
 
-function mysqli_stmt_data_seek(mysqli_stmt $mysql_stmt, int $offset): void {}
+function mysqli_stmt_data_seek(mysqli_stmt $stmt, int $offset): void {}
 
-function mysqli_stmt_errno(mysqli_stmt $mysql_stmt): int {}
+function mysqli_stmt_errno(mysqli_stmt $stmt): int {}
 
-function mysqli_stmt_error(mysqli_stmt $mysql_stmt): ?string {}
+function mysqli_stmt_error(mysqli_stmt $stmt): ?string {}
 
-function mysqli_stmt_error_list(mysqli_stmt $mysql_stmt): array {}
+function mysqli_stmt_error_list(mysqli_stmt $stmt): array {}
 
-function mysqli_stmt_fetch(mysqli_stmt $mysql_stmt): ?bool {}
+function mysqli_stmt_fetch(mysqli_stmt $stmt): ?bool {}
 
-function mysqli_stmt_field_count(mysqli_stmt $mysql_stmt): int {}
+function mysqli_stmt_field_count(mysqli_stmt $stmt): int {}
 
-function mysqli_stmt_free_result(mysqli_stmt $mysql_stmt): void {}
+function mysqli_stmt_free_result(mysqli_stmt $stmt): void {}
 
 #if defined(MYSQLI_USE_MYSQLND)
-function mysqli_stmt_get_result(mysqli_stmt $mysql_stmt): mysqli_result|false {}
+function mysqli_stmt_get_result(mysqli_stmt $stmt): mysqli_result|false {}
 #endif
 
-function mysqli_stmt_get_warnings(mysqli_stmt $mysql_stmt): mysqli_warning|false {}
+function mysqli_stmt_get_warnings(mysqli_stmt $stmt): mysqli_warning|false {}
 
-function mysqli_stmt_init(mysqli $mysql_link): mysqli_stmt|false {}
+function mysqli_stmt_init(mysqli $connection): mysqli_stmt|false {}
 
-function mysqli_stmt_insert_id(mysqli_stmt $mysql_stmt): int|string {}
+function mysqli_stmt_insert_id(mysqli_stmt $stmt): int|string {}
 
 #if defined(MYSQLI_USE_MYSQLND)
-function mysqli_stmt_more_results(mysqli_stmt $mysql_stmt): bool {}
+function mysqli_stmt_more_results(mysqli_stmt $stmt): bool {}
 
-function mysqli_stmt_next_result(mysqli_stmt $mysql_stmt): bool {}
+function mysqli_stmt_next_result(mysqli_stmt $stmt): bool {}
 #endif
 
-function mysqli_stmt_num_rows(mysqli_stmt $mysql_stmt): int|string {}
+function mysqli_stmt_num_rows(mysqli_stmt $stmt): int|string {}
 
-function mysqli_stmt_param_count(mysqli_stmt $mysql_stmt): int {}
+function mysqli_stmt_param_count(mysqli_stmt $stmt): int {}
 
-function mysqli_stmt_prepare(mysqli_stmt $mysql_stmt, string $query): bool {}
+function mysqli_stmt_prepare(mysqli_stmt $stmt, string $query): bool {}
 
-function mysqli_stmt_reset(mysqli_stmt $mysql_stmt): bool {}
+function mysqli_stmt_reset(mysqli_stmt $stmt): bool {}
 
-function mysqli_stmt_result_metadata(mysqli_stmt $mysql_stmt): mysqli_result|false {}
+function mysqli_stmt_result_metadata(mysqli_stmt $stmt): mysqli_result|false {}
 
-function mysqli_stmt_send_long_data(mysqli_stmt $mysql_stmt, int $param_nr, string $data): bool {}
+function mysqli_stmt_send_long_data(mysqli_stmt $stmt, int $index, string $data): bool {}
```

Comment:
I would say that it is already clear from the type that `$index` is the index of a parameter. My preference would be to call it `$parameter`.